tailwind.config.js

```diff
@@ -3,14 +3,19 @@ module.exports = {
   darkMode: false, // or 'media' or 'class'
   theme: {
     extend: {
+      dropShadow: {
+        '3xl': '0 100px 100px rgba(0, 0, 0, 0.75)',
+      },
       colors: {
         'line-divide-color': '#2a3641',
         'link-orange': '#ff9956',
         'card-background-color' : '#323f4b',
+        'card-dark-background-color' : '#19191a',
         'card-even-color' : '#2a3641',
         'card-border-color' : '#3871c2',
         'card-header-text-color' : '#7b8794',
-        'dark-block-color' : '#1F2933'
+        'dark-block-color' : '#1F2933',
+        'border-dark-color' : "#141415"
       }
     },
   },
```

Comment:
Ditto on coordinating with Roxas re: changes here – I'd rather have both of you be happy than me be miserable sorting it out :-)

Reply:
@OHaiDanny once my PR is merged I can take care of these colors in your PR as well. We most likely used the same color codes but with different naming. I'm more a fan of not having words like "color" inside of color-variables since writing them unser color imply that they serve as color suffixes.

Reply:
I've updated the colors to use what @Roxas92 merged in :)